apps/api/src/app/environments/novu-nestjs-handler.ts

```diff
@@ -0,0 +1,88 @@
+import type { Request, Response } from 'express';
+import { NovuRequestHandler, ServeHandlerOptions, Either, SupportedFrameworkName } from '@novu/framework';
+
+export const frameworkName = 'nestjs';
+
+export class NovuNestjsHandler {
+  private novuHandler: NovuRequestHandler;
+
+  constructor(options: ServeHandlerOptions) {
+    this.novuHandler = new NovuRequestHandler({
+      frameworkName,
+      ...options,
+      handler: this.handler.bind(this),
+    });
+  }
+
+  private handler(requestMethod: 'GET' | 'POST' | 'OPTIONS' | undefined, incomingRequest: Request, response: Response) {
```

Comment:
Nice!
💡 In this cycle, we have to create the NestJS handler of @novu/framework. Please extract this as the Nest.js handler, add it to @novu/framework and then dog food.